presto-jdbc/src/main/java/com/facebook/presto/jdbc/PrestoStatement.java

```diff
@@ -194,6 +199,15 @@ public void setCursorName(String name)
         // ignore: positioned modifications not supported
     }
 
+    private Map<String, String> getStatementSessionProperties() {
```

Comment:
The "statement" word in here is redundant for an internal, private method. This can be `getSessionProperties()`